codalab/server/bundle_manager.py

```diff
@@ -329,8 +322,6 @@ def _schedule_run_bundles_on_workers(self, workers, user_owned):
             for worker in workers_list:
                 if self._try_start_bundle(workers, worker, bundle):
                     break
-                else:
-                    continue  # Try the next worker.
```

Comment:
This code doesn't do anything since the for loop naturally continues at this point.